web/src/main.tsx

```diff
@@ -171,23 +183,11 @@ const router = createBrowserRouter([
       },
       {
         path: '/map',
-        element: (
-          <Navigate
-            to="/map/24h"
-            replace
-            state={{ preserveSearch: true, preserveHash: true }}
-          />
-        ),
+        element: <Navigate to="/map/24h" replace />,
       },
       {
         path: '/zone',
-        element: (
-          <Navigate
-            to="/map/24h"
-            replace
-            state={{ preserveSearch: true, preserveHash: true }}
```

Comment:
I assume these are the defaults?

Reply:
It's to write to [the url state](https://developer.mozilla.org/en-US/docs/Web/API/History/state)

Which in this case we don't even use those keys. Cursor added it at some point and I thought it seemed logical but looking deeper it does nothing!